traffic_ops/traffic_ops_golang/riak.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/tostructs"
+	"github.com/basho/riak-go-client"
+	"github.com/jmoiron/sqlx"
+	"github.com/lestrrat/go-jwx/jwk"
+	"io/ioutil"
+	"net/http"
+)
+
+const riak_port = 8087
```

Comment:
Go Constants should be named the same as other global symbols, with MixedCase. See https://golang.org/doc/effective_go.html#mixed-caps